### ISSUE TITLE
Panic runtime error for select issuers transaction

### ISSUE THREAD

**Issue by jmrossy** [celo-org/celo-blockchain#661]
There's a particular transaction with the new off-chain reveal verification scheme, the select issuers tx, that triggers a geth panic runtime error about half the time. I'm testing with the latest geth built from celo-blockchain master on an android phone.

Logs here:

[geth panic.txt](https://github.com/celo-org/celo-blockchain/files/3897917/geth.panic.txt)

@asaj believes it's probably related to the precompile that gets you the validator set

**Comment by nategraf** [comment]
We investigated this issue last Wednesday. We attempted to reproduce this issue by running `selectIssuers` transactions through an `ultralight` client attached to a local testnet and to `baklava-staging`, which approximates the setup for phones. The bug failed to reproduce in those tests or subsequent builds of the mobile app.

Because we have not seen it again, I am closing this issue and will reopen it if the issue is seen again.
